src/openms/include/OpenMS/FILTERING/NOISEESTIMATION/SignalToNoiseEstimatorMedian.h

```diff
@@ -311,13 +317,14 @@ namespace OpenMS
       double noise;    // noise value of a datapoint
 
       // determine how many elements we need to estimate (for progress estimation)
-      int windows_overall = 0;
-      PeakIterator run = scan_first_;
-      while (run != scan_last_)
-      {
-        ++windows_overall;
-        ++run;
-      }
+//      int windows_overall = 0;
+//      PeakIterator run = scan_first_;
+//      while (run != scan_last_)
+//      {
+//        ++windows_overall;
+//        ++run;
+//      }
+      int windows_overall = c.size();
```

Comment:
`int windows_overall` is only used once here - neccessary?

Reply:
It can be used directly in the function . If we used it, it can makes code more clear and understandable.